build/scripts/yarn-scripts.ts

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env node
```

Comment:
Should this go before or after the license blurb?

I've also never seen a _shebang_ for a `.ts` file. Is it necessary?